CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Change History
 **************
 
+2.13.x (unreleased)
```

Comment:
This code is targeting master, so this should be 3.0.0a4, shouldn't it? 

If we want a 2.x release (IMO, we don't—3.0 is the way forward) this would need to be cherry-picked back to a new branch.

Reply:
I had a look around the branches and the developer documentation and I found no clue how the development will go forward concerning the version 2 and the version 3 development line.

Just last week @icemac released a new 2.13 version. @icemac Can you give us a hint? On which branch did you develop?

The version 3 line seems to be stalled for almost a complete year now, and also there seem to be at least a couple of reported problems, see #510 and #515

Also, I am not sure whether the alpha releases are already well tested in the field, e.g. by Plone and maybe some Zope projects - usually a lot of edge cases only surface when tested against real projects.

Also, will version 3 still support Python 2.7 or is this the moment for the cut? I have no clue.

As I use mr. developer and there seems to be a problem in the version 3 (see above #515), I'd be happy if there would be a new 2.13 release - except that would be too much effort.

Reply:
I use buildout 3.0 with a very large Zope 3 project, mr.developer, and a relatively complicated buildout, and haven't run into those specific problems. (Granted, that's just my local dev environment; we're not using it in production yet.)

I think 3.0 should/will support 2.7: 2.7 users need a path forward (and 2.7 is what I've been using 3.0 with).

The problem with pre-releases is that almost nobody tests them, since pip stopped installing them. The only way to get real testing, aside from begging the community, is to release a final version.

Reply:
Thanks for your time - will be fixed with the next commit.

Reply:
The last `zc.buildout` release is about a year old: https://pypi.org/project/zc.buildout/2.13.3/
There is the branch https://github.com/buildout/buildout/tree/2.x – I think the 2.13.x development shout happen there. – but probably we also need your changes on `master`, too.

Reply:
Thanks for the correction - seems I still live in 2020 :-D Oh this pandemic!

master was def. broken, so imho yes, we need the changes here.

I will have a look at the 2.x branch (where dev.py looks completely different), once this got merged and/or I got the right to merge myself.

@icemac Maybe you can enable me to get forward here.